### PR TITLE
Add functionality to remove a single locale from a page

### PR DESCRIPTION
**Pull request by Prokyonn**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | yes
| Deprecations? | no
| Fixed tickets | fixes https://github.com/sulu/sulu/issues/431
| Related issues/PRs | #issuenum
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Adjusts the `DeleteToolbarAction` for forms to delete only a specific locale.
Adds the backend functionality to delete only a specific locale of a page (removes all localized properties for this locale)

#### BC Breaks/Deprecations

`DocumentManagerInterface` has been extended with a `removeLocale($document, $locale)` method.

#### To Do

- [ ] Create a documentation PR
- [ ] Add breaking changes to UPGRADE.md

### CODE REVIEW

File: src/Sulu/Component/DocumentManager/Event/RemoveLocaleEvent.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    public function __construct($document, $locale)
+    {
+        $this->document = $document;
+        $this->locale = $locale;
+    }
+
+    /**
+     * Sets the node this event should operate on.
+     *
+     * TODO Check if should be move to DocumentManager itself
```
Comment (wachterjohannes) [inline src/Sulu/Component/DocumentManager/Event/RemoveLocaleEvent.php:31]: for other events we also set the node from a subscriber - so feel free to remove this comment :)

**Review by wachterjohannes — APPROVED** [review]

**Comment by wachterjohannes** [comment]
@Prokyonn remove the comment and then we can merge this PR

File: src/Sulu/Component/DocumentManager/Event/RemoveLocaleEvent.php (hunk 1 above)
Comment (Prokyonn) [reply src/Sulu/Component/DocumentManager/Event/RemoveLocaleEvent.php:31]: Yeah but on these events there is the same comment, should I remove all of them?

File: src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/DeleteToolbarAction.js [hunk 2]
```diff
@@ -27,12 +27,13 @@ export default class DeleteToolbarAction extends AbstractFormToolbarAction {
         form: Form,
         router: Router,
         locales: ?Array<string>,
-        options: {[key: string]: mixed},
+        options: { [key: string]: mixed },
```
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/DeleteToolbarAction.js:30]: That's not how we usually write that... Is there maybe a ESLint rule missing resp. not working anymore? :thinking:

**Pull request merged**